{"uuid": "660b3db7-29ce-4d94-8e6d-24bc5b806faf", "children": ["8154f2d1-04e9-4b24-9c27-a6b9262928dc"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "Exception: 无该校验方式：no_check\n", "trace": "  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/nali/AutoApiTest/conftest.py\", line 28, in login\n    api_send_check(request.param, case_dict, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/api_send_check.py\", line 33, in api_send_check\n    check_result.check_result(test_name, check_data, code, data, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/check_result.py\", line 128, in check_result\n    raise Exception(\"无该校验方式：{}\".format(case[\"check_type\"]))\n"}, "steps": [{"name": "POST请求接口", "status": "passed", "attachments": [{"name": "请求接口：", "source": "5ec1b041-31fa-4ee2-8233-323c401b1243-attachment.attach"}, {"name": "用例描述：", "source": "d36ba43c-1778-41e7-b875-f39bdf6a7009-attachment.attach"}, {"name": "请求地址", "source": "323db1e5-3785-4c77-945d-c7babc9b3019-attachment.attach"}, {"name": "请求头", "source": "c3ce6055-e5ac-4ba5-8278-9f27f4c0773b-attachment.attach"}, {"name": "请求参数", "source": "be6d5b87-7b34-4643-be66-45744a6b5bab-attachment.attach"}], "start": 1617289782081, "stop": 1617289782082}, {"name": "不校验结果", "status": "passed", "start": 1617289782334, "stop": 1617289782334}], "start": 1617289782078, "stop": 1617289782334}], "start": 1617289782078, "stop": 1617289782354}